src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/CountFunctions.txt

```diff
@@ -23,4 +23,7 @@ Blank()
 #Error
 
 >> CountIf([1, 2, 3, 4], 123)
-Errors: Error 22-25: Invalid argument type (Number). Expecting a Boolean value instead.
+Errors: Error 22-25: Expected boolean. We expect a boolean (true/false) at this point in the formula.
+
+>> CountIf([1, 2, 3, 4], "abc")
+Errors: Error 22-27: Expected boolean. We expect a boolean (true/false) at this point in the formula.
```

Comment:
This is odd, where else is this error used? It makes me think it expects a boolean literal, but that's not the case. The InvalidArgumentType Error should be good enough here